ui/app/AppLayouts/Profile/views/WalletView.qml

```diff
@@ -81,24 +84,29 @@ SettingsContentBase {
         }
 
         NetworksView {
-            walletStore: root.walletStore
+            layer1Networks: root.walletStore.layer1Networks
+            layer2Networks: root.walletStore.layer2Networks
```

Comment:
Same here... all our components are a properties hell :) Having the logic encapsulated in a store makes it at least a bit more manageable

Reply:
IMO this making this explicit is just far superior, walletStore is just encapsulating and hidding things making them much harder to understand, also ends up passing all this javascript around for components that don't even need it